app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -210,6 +210,7 @@ public void setColorFilter(ColorFilter cf) {}
         previewChart.setZoomType(ZoomType.HORIZONTAL);
 
         chart.setLineChartData(bgGraphBuilder.lineData());
+        chart.setOnValueTouchListener(bgGraphBuilder.getOnValueSelectTooltipListener(this));
```

Comment:
Passing around activities is a good way to leak large amounts of RAM.  I would feel better about it if it instantiated a new anonymous listener and passed that instead.